components/header/HeaderNavigationButton.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Typography, useTheme } from '@mui/material';
+import Link from 'next/link';
+import { PropsWithChildren } from 'react';
+
+interface HeaderNavigationButtonProps extends PropsWithChildren {
+  link: string;
+}
+
+const HeaderNavigationButton = ({
```

Comment:
**nitpick:** If we change to use a `nav` to wrap these but we can probably stick with the html tag name of `nav` to keep this clear that it is a `<nav>` of the button style, especially if we use a `<nav>` to wrap it - so we can get away with `HeaderNavButton`.

Reply:
Done